### DISCUSSION TITLE
Feature Request: Add "Add to Folder" option in Upload Options dialogue

### DISCUSSION THREAD

**Discussion by jordanlambrecht** [diced/zipline#558] [Ideas]
It would be great if there was an option to directly add uploaded images to a folder via this dialogue:

<img width="629" alt="Screenshot 2024-06-04 at 10 35 46" src="https://github.com/diced/zipline/assets/54866853/bc8a3d27-d5e3-45c8-bf5a-40a2ca6303aa">

**Comment by ItsMelodyy0102** [comment]
This would be fantastic. Definitely a QoL thing that's missing
